core/src/main/resources/lib/hudson/project/console-link.jelly

```diff
@@ -28,11 +28,11 @@ THE SOFTWARE.
     <j:choose>
         <j:when test="${it.logFile.length() > 200000}">
             <!-- Show raw link directly so user need not click through live console page, though this is not so bad now as they would just see: Skipping nnn KB.. Full Log. -->
-            <l:task href="${buildUrl.baseUrl}/console" icon="icon-terminal icon-md" title="${%Console Output}"/>
+            <l:task href="${h.getConsoleUrl(it)}" icon="icon-terminal icon-md" title="${%Console Output}"/>
             <l:task href="${buildUrl.baseUrl}/consoleText" icon="icon-document icon-md" title="${%View as plain text}"/>
```

Comment:
I am not entirely sure what we want to do about this case. I think it makes sense to redirect the main "console output" link, but IDK if we should keep the "View as plain text" link or hide it if the console URL is non-default or something like that. For now I am inclined to just leave it as-is.